sonarqube/assets/configuration/spec.yaml

```diff
@@ -89,6 +93,57 @@ files:
             type: array
             items:
               type: string
+      - name: projects
+        description: |
+          The projects for which metrics should be collected. Each object may override the default
+          options. The following example gathers all default metrics for "some-project",
+          and only what is included/excluded for "another-project":
+          
+            projects:
+              keys:
+                - some-project:
+                - another-project:
+                    tag: project-2
+                    metrics:
+                      discovery:
+                        include:
+                        - issues\..*
+                        - maintainability\..*
+                        - security\..*
+                        exclude:
+                        - issues.(false_positive_issues|reopened_issues)
+              discovery:
+                exclude:
+                - tmp_project\..*
+          
+          Note: This is required for all project checks, whether `is_jmx` is set to `true`
```

Comment:
I don't understand this part. If `is_jmx` is set to true none of this applies :/

Reply:
Result of copy&paste...
If I remove here, I would have to delete from components key too.
It sounded weird to me too when I saw it.

Reply:
yes, I don't think it makes sense in components either

Reply:
@ofek do you have more context?

Reply:
Updated the Note to: `Note: This only takes effect when 'is_jmx' is set to 'false'.` as in the rest of the document